### ISSUE TITLE
Implement handling multi destination

### ISSUE THREAD

**Issue by jhlee-mitre** [fhir-crucible/testscript-engine#53]
**Goal**
Implemented a feature to handle multi-destinations in TestScript

**FHIR Spec**
https://www.hl7.org/fhir/testscript-definitions.html#TestScript.destination

**Detail**
There are two elements in TestScript concern multi actor: origin and destination. Since the engine is a command line based application, origins may be the engine itself regardless. Therefore, the engine only concerns destination. The engine reads TestScript when it is loaded and figure out the number of destination. If there are multiple destinations, the engine will ask a user to enter server addresses for each corresponds to destination. 

**Testscript**
Example TestScript for multi-agent should be included.

**Comment by Jack-Fraser16** [comment]
This should probably entail creating a client class as well, which will allow us to keep better track of the different clients we have running around.

**Comment by jhlee-mitre** [comment]
> This should probably entail creating a client class as well, which will allow us to keep better track of the different clients we have running around.

I agree with you for the approach. Should it be included in the first phase of multi actor (one origin multiple destination) or we can approach step by step maybe?
